src/Sulu/Bundle/SecurityBundle/Resources/public/js/components/users/components/form/main.js

```diff
@@ -232,6 +233,16 @@ define(['app-config', 'widget-groups'], function(AppConfig, WidgetGroups) {
             ]);
         },
 
+        bindFormEvents: function() {
+            this.sandbox.dom.one(this.$el, 'focus', function() {
+                this.sandbox.dom.val('input[type="password"]', '');
+            }.bind(this), 'input[type="password"]');
+
+            this.sandbox.dom.one(this.$el, 'keyup', function() {
+                this.sandbox.dom.data('#password', 'changed', true);
```

Comment:
@marcelmoosbrugger the user cant set his passwort to the placeholder value that way. i think the changed-state of the input is the only way to do this without excluding possible password